influxdb3/query.go

```diff
@@ -87,21 +108,28 @@ func (c *Client) Query(ctx context.Context, query string) (*QueryIterator, error
 // Returns:
 //   - A custom iterator (*QueryIterator) that can also be used to get raw flightsql reader.
 //   - An error, if any.
+//
+// Deprecated: use Query with QueryOption options.
```

Comment:
I think the Deprecation message could be more precise. e.g. "Deprecated: use Query with variadic QueryOption options".  The deprecated method itself has a similar argument with a pointer to QueryOptions.  At first glance it can be a bit confusing.

Reply:
The message is updated.